Jenkinsfile

```diff
@@ -1,22 +1,22 @@
 env.LANG="en_US.UTF-8"
 env.LANGUAGE="en_US.UTF-8"
 env.LC_ALL="en_US.UTF-8"
+
 node ('macos1') {
   def apkUrl = ''
   def ipaUrl = ''
   def testPassed = true
   def branch;
 
-   load "$HOME/env.groovy"
+  load "$HOME/env.groovy"
 
   try {
 
     stage('Git & Dependencies') {
       slackSend color: 'good', message: BRANCH_NAME + ' build started. ' + env.BUILD_URL
+
       git([url: 'https://github.com/status-im/status-react.git', branch: BRANCH_NAME])
-      // Checkout master because used for iOS Plist version information
-      sh 'git checkout -- .'
```

Comment:
ℹ️ This is the relevant change I made to this file – I think we don't need to do this checkout swap anymore. I can't test on Jenkins without being a core contrib though so I'll need help on this.

Otherwise, the edits to this file were me auto-fixing the indents, as they were all over the place.

Reply:
We also have Jenkinsfile.nightly and Jenkinsfile.parameters (recent/experimental), FYI

Reply:
Thanks, that one is still in PR I realized

Reply:
Should we leave an "FYI" comment on that PR?

Reply:
Haha I literally just did! https://github.com/status-im/status-react/pull/2564#issuecomment-349397940